cherrypy/process/wspbus.py

```diff
@@ -342,15 +342,10 @@ def block(self, interval=0.1):
             # that another thread executes cherrypy.engine.exit()
             if (
                     t != threading.currentThread() and
-                    t.isAlive() and
                     not isinstance(t, threading._MainThread)
```

Comment:
Now there is no AssertionError . Do you think we need `threading._DummyThread` check? I can add it to if condition again.